app/middleware/cache_headers.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from django.conf import settings
+from django.utils.cache import add_never_cache_headers
+from django.utils.cache import patch_cache_control
+from django.utils.cache import patch_response_headers
+
+logger = logging.getLogger(__name__)
+
+
+class CacheHeadersMiddleware:
+    '''Middleware that adds appropriate cache headers to GET and HEAD methods.
+
+    NOTE: /checker and /get-token endpoints are marked as never cache.
+    '''
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        # Code to be executed for each request before
+        # the view (and later middleware) are called.
+
+        response = self.get_response(request)
+
+        # Code to be executed for each request/response after
+        # the view is called.
+
+        if request.path in ['/checker', '/get-token']:
```

Comment:
What about the admin interface? Would it also make sense not to cache that one, too?

Reply:
Admin interface are not cached. The middleware don't act on the admin interface, but only on the Django views. For the Admin interface the statics files are then cached via Whitenoise while the admin api is not cached (django adds never cache headers)